packages/client/tests/functional/create-many/prisma/_schema.ts

```diff
@@ -0,0 +1,30 @@
+import { idForProvider } from '../../_utils/idForProvider'
+import testMatrix from '../_matrix'
+
+export default testMatrix.setupSchema(({ provider, previewFeatures }) => {
+  return /* Prisma */ `
+    generator client {
+      provider = "prisma-client-js"
+      previewFeatures = [${previewFeatures}]
+    }
+    
+    datasource db {
+      provider = "${provider}"
+      url      = env("DATABASE_URI_${provider}")
+    }
+    
+    model User {
+      id ${idForProvider(provider)}
```

Comment:
Small nit here about the indentation.

Reply:
https://github.com/prisma/prisma/pull/13498/commits/ce59f5b9820d8a5459c90115b6c0fff978451972